### PR TITLE
qt/qt5: update qt links to new urls

### PR DESCRIPTION
**Pull request by DomT4**
More detailed comments in the individual commits.

### CODE REVIEW

**Comment by DomT4** [comment]
@mikemcquaid I was hoping I could just kill the build here to stop the bot going wild on an unnecessary full-rebuild of the two qt formulae, but I can't. Lesson learnt. Apologies for the bot stress :cry:

File: Library/Formula/qt.rb [hunk 1]
```diff
@@ -1,12 +1,10 @@
-require 'formula'
-
 class Qt < Formula
   homepage "http://qt-project.org/"
 
   stable do
-    # Mirror rather than source set as primary because source is very slow.
+    # Mirror rather than source as primary because source is vslow. New .io address seems much quicker, switch back?
```
Comment (mikemcquaid) [inline Library/Formula/qt.rb]: Yeh, let's switch back.

Comment (DomT4) [inline Library/Formula/qt.rb]: Switched back over. May want to stop the bot going wild here again.

**Comment by DomT4** [comment]
Reopened following discussion with Mike. Will rebase against the master branch shortly to resolve the merge conflict.

**Comment by DomT4** [comment]
Oops, Hold on that. I can't reopen because I can't override Mike's close :smile_cat:. I'll open a new PR.

**Pull request closed without merging**